src/environment_setup/biopam.ml

```diff
@@ -4,172 +4,250 @@ Provide tools via Biopam: https://github.com/solvuu/biopam
 
 open Biokepi_run_environment
 open Common
-module K = KEDSL
 
 (* What are we installing via opam. This determines where we look for the
    witness; in [opam_install_path]/package or [opam_install_path]/bin. *)
-type tool_type =
-  | Library of string
-  | Application
+type tool_type = [
```

Comment:
Yes, but poly-variants are nicer in APIs (no name-clash problems). And regular variants don't bring anything better.